tgui/packages/tgui/interfaces/ChemDispenser.js

```diff
@@ -57,7 +57,7 @@ export const ReagentDispenser = (props, context) => {
     maximumBeakerVolume,
     beakerTotalVolume,
   } = data;
-  const [addAmount, setAddAmount] = useLocalState(context, 'addAmount', 10);
+  const [addAmount, setAddAmount] = useSharedState(context, 'addAmount', 10);
```

Comment:
This, and equivalent for the remove amount, are the only "real" changes. Rest is just formatting. Sue me.